sys-cluster/easybuild/.build

```diff
@@ -0,0 +1,7 @@
+go
+a
+b
+c
```

Comment:
This looks like a leftover file that we don't need in the repository.

Reply:
Whoops, thanks. I have a second local .gitignore now masking my remnant .build files.